CPP_fibonacci.cpp

```diff
@@ -0,0 +1,22 @@
+// Fibonacci upto nth term (Iterative Method)
+#include<bits/stdc++.h>
+using namespace std;
+void get_fibb(long int n)
+{
+	long int f0=0,f1=1;
+	if(n<1)
+		return;
+	
+	for(long int i=1;i<=n;i++){
+		cout<<f1<<" ";
+		long int tmp=f0+f1;
+		f0=f1;
+		f1=tmp;
+	}
+}
+int main()
+{
+	long int n;
+	cin>>n;
+	get_fibb(n);
+}
```

Comment:
Hey @bluecoder5015 , I appreciate your enormous contribution.  But the code doesn't meet required expectations. For more details Please refer to issue no. #9 . 

Looking forward to see your beautiful code with updated changes.

Keep Contributing!!
Happy Coding!!